### ISSUE TITLE
Incorrect shape size for NN input data

### ISSUE THREAD

**Issue by RK900** [Eternagame/EternaBrain#9]
### Tensorflow
```python
cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=prediction,labels=y))
```
Logits and labels not of same size.

### TFLearn
```python
net = tflearn.input_data(shape=[None, 4716])
```
Shape size is incorrect for input matrix.
